bin/circle_rubocop.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
```

Comment:
Based on comments above I think this script can be removed. Also it might have other rubocop offenses with ruby 2.4.

Reply:
See my comments above... left it for now.